build/wd_test.bzl

```diff
@@ -0,0 +1,59 @@
+def wd_test(
+        conf,
+        srcs = [],
+        name = None,
+        flags = []):
+    """Rule to define tests that run `workerd test` with a particular config.
+
+    Args:
+     conf: A .capnp config file defining the test. (`name` will be derived from this if not
+        specified.)
+     srcs: Files which the .capnp config file may embed. Typically JavaScript files.
```

Comment:
data is more common name for this attribute

Reply:
I can change it to that if you prefer. My thinking was that this will almost always be used to specify the JavaScript source files, so it feels more natural as `srcs` from the user's perspective, but you're right that `data` seems more correct from the build system's perspective.

Reply:
I went ahead and changed all the property names to be more standard, and added `**kwargs` so other properties should work too.